Makefile

```diff
@@ -197,7 +197,11 @@ RegEx: 3rdParty/RegEx/OMSRegEx$(EEXT)
 3rdParty/RegEx/OMSRegEx$(EEXT): 3rdParty/RegEx/RegEx.h 3rdParty/RegEx/OMSRegEx.cpp
 	$(MAKE) -C 3rdParty/RegEx
 
-config-3rdParty: config-fmil config-lua config-zlib config-cvode config-kinsol config-ceres-solver config-libxml2
+3rdParty/README.md:
```

Comment:
Initially, I've been using `3rdPart/.git`, but that might not always be available.